### ISSUE TITLE
Auto Approval based on Release Status

### ISSUE THREAD

**Issue by JacobBrown4** [petio-team/petio#346]
**Is your feature request related to a problem? Please describe.**
I usually run auto approval for movies for users and do tv manually. This is mostly just me being particular about downloading gigs of files for a `150 ep TV show unexpectedly.

**Describe the solution you'd like**
An option to auto approve TV and Movies by release type, eg. Announced, Airing, etc. That way I can have TV show's that haven't started yet auto approved.

**Describe alternatives you've considered**
Adding an auto approval option to the filter options for matching content. Then adding release status as a filter. (I think this is the stronger option since it'll let people stack filters and auto approval for certain things.
An approval threshold for episodes aired would be nice as well. (Ex. Less than three episodes have come out for a show, go ahead and auto approve it)

**Comment by ADRFranklin** [comment]
I think it's possible for us to improve the auto approve system to allow for such an option, but I don't think it is something that will happen like what you have described, however it would have to be discussed between the developers to workout a proper way to make this happen. 

It's not something we are planning to do just yet, but is something we can get around to later down the line.

**Comment by AshDyson** [comment]
yes seconded, we can allow auto approve conditions, but it's not on the immediate roadmap, will leave open for now
